packages/agents/router/src/subscriber/bindings/exceptions/index.ts

```diff
@@ -0,0 +1,15 @@
+import { createLoggingContext } from "@connext/nxtp-utils";
+
+import { getContext } from "../../subscriber";
+
+export const bindUnhandledErrors = async () => {
+  const { logger } = getContext();
+  const { requestContext, methodContext } = createLoggingContext(bindUnhandledErrors.name);
+  process.on("unhandledRejection", (error) => {
+    logger.warn("Unhandled rejection occured!", requestContext, methodContext, { error });
+  });
+
+  process.on("uncaughtException", (error) => {
```

Comment:
This is very dangerous to do. We don't want to catch all unhandled errors, we specifically handle errors with try catch if we want them caught.